src/Features/JsonPatch.SystemTextJson/src/JsonPatchDocument.cs

```diff
@@ -218,4 +221,19 @@ IList<Operation> IJsonPatchDocument.GetOperations()
 
         return allOps;
     }
+
+    /// <summary>
+    /// Populates metadata for the related endpoint when this type is used as a parameter.
+    /// </summary>
+    /// <param name="parameter">The <see cref="ParameterInfo"/> for the endpoint parameter.</param>
+    /// <param name="builder">The endpoint builder for the endpoint being constructed.</param>
+#pragma warning disable RS0016 // Add public types and members to the declared API
+    public static void PopulateMetadata(ParameterInfo parameter, EndpointBuilder builder)
+#pragma warning restore RS0016 // Add public types and members to the declared API
+    {
+        ArgumentNullException.ThrowIfNull(parameter);
+        ArgumentNullException.ThrowIfNull(builder);
+
+        builder.Metadata.Add(new AcceptsMetadata(new[] { "application/json-patch+json" }, parameter.ParameterType));
```

Comment:
```suggestion
        builder.Metadata.Add(new AcceptsMetadata(["application/json-patch+json"], parameter.ParameterType));
```

Here and elsewhere, use collection expressions instead of arrays.